setup.cfg

```diff
@@ -1,3 +1,43 @@
 [flake8]
 max-line-length = 88
 extend-ignore = E203
+exclude = build, ci-helpers
+ignore =
+
+[tool:pytest]
```

Comment:
let's put this stuff in `pyproject.toml` instead, ideally we'd get rid of `setup.cfg`

Reply:
I just tried to move the codespell configuration to pyproject.toml, but it only understands the configuration in setup.cfg.

```
[tool.codespell]
builtin=clear,rare,informal,usage,code,names
ignore-words-list=uint
skip=./.*,codespell_lib/data/*,codespell_lib/tests/*

```

Reply:
mypy provides guidance about pyproject.toml, but this pyproject.toml configuration was not picked up by codespell either:

```
[tool.codespell]
builtin=["clear","rare","informal","usage","code","names"]
ignore-words-list="uint"
skip=["./.*","codespell_lib/data/*","codespell_lib/tests/*"]
```

Reply:
The additional dependency 'tomli' is required:
```yaml
  - repo: https://github.com/codespell-project/codespell
    rev: v2.2.2
    hooks:
      - id: codespell
        additional_dependencies:
          - tomli
```

codespell now complains:
```
ERROR: Unknown builtin dictionary:  'code'
```

With the following toml configuration:
```
[tool.codespell]
builtin = ["clear","rare","informal","usage","code","names"]
ignore-words-list = ["uint"]
skip=[ "./.*","codespell_lib/data/*","codespell_lib/tests/*"]
```

It did not complain about that with the setup.cfg configuration:
```
[codespell]
builtin=clear,rare,informal,usage,code,names
ignore-words-list=uint
skip=./.*,codespell_lib/data/*,codespell_lib/tests/*
```

Reply:
Actually, codespell does not support the lists in pyproject.toml while other tools do ..
This is understood:
```
[tool.codespell]
#builtin = ["clear","rare","informal","usage","code","names"]
builtin = "clear,rare,informal,usage,code,names"
#ignore-words-list = ["uint"]
ignore-words-list = "uint"
#skip=[ "./.*","codespell_lib/data/*","codespell_lib/tests/*"]
skip="./.*,codespell_lib/data/*,codespell_lib/tests/*"
```

Reply:
> Actually, codespell does not support the lists in pyproject.toml while other tools do ..

Feel free to improve the support here if you want. But it'll require new test cases and debugging, so maybe better in a separate PR

I pushed a commit to fix `make check-manifest`